packages/map/src/map/hooks.jsx

```diff
@@ -34,12 +34,17 @@ export const useMapEffect = (fn, deps = []) => {
 
 export const useMapInstance = (nodeRef, options) => {
   const [mapInstance, setMap] = useState(false);
+  const [webGLSupport, setWebGLSupport] = useState(true);
```

Comment:
also probably setting it to `false` by default and enabling later would be better, example:

```
const useWebGLStatus = () => {
  // Probably setting to `false` by default is better to avoid crashes
  const [isSupported, setSupported] = useState(false);
  useEffect(() => { setSupported(isWebGLSupported(global.document)); }, []);
  return isSupported;
}
```